packages/newsroom-signup/src/actionCreators.ts

```diff
@@ -8,6 +8,7 @@ import { makeUserObject } from "./utils";
 
 export enum newsroomActions {
   ADD_NEWSROOM = "ADD_NEWSROOM",
+  ADD_MULTISIG = "ADD_MULTISIG",
```

Comment:
Is this redundant with the `SET_MULTISIG_ADDRESS` action that jon's PR added?

Reply:
yeah